workload_generator/src/workload_gen.py

```diff
@@ -123,75 +130,74 @@ def sendWorkload(self, user, pid):
 
             if command == BUY:
                 (stockSymbol, amount) = (requestInfo[2], float(requestInfo[3].strip("\n")))
-                self.performRequest(pid, transactionNumber, command, user, stockSymbol, amount)
+                self.performRequest(url1, transactionNumber, command, user, stockSymbol, amount)
 
             elif command == SELL:
                 (stockSymbol, amount) = (requestInfo[2], float(requestInfo[3].strip("\n")))
-                self.performRequest(pid, transactionNumber, command, user, stockSymbol, amount)
+                self.performRequest(url1, transactionNumber, command, user, stockSymbol, amount)
 
             elif command == SET_BUY_AMOUNT:
                 (stockSymbol, amount) = (requestInfo[2], float(requestInfo[3].strip("\n")))
-                self.performRequest(pid, transactionNumber, command, user, stockSymbol, amount)
+                self.performRequest(url1, transactionNumber, command, user, stockSymbol, amount)
 
             elif command == SET_BUY_TRIGGER :
                 (stockSymbol, amount) = (requestInfo[2], float(requestInfo[3].strip("\n")))
-                self.performRequest(pid, transactionNumber, command, user, stockSymbol, amount)
+                self.performRequest(url1, transactionNumber, command, user, stockSymbol, amount)
```

Comment:
if triggers are in new url location, it will need to be added and changed here. 

lmk sam.

Reply:
Yes it is in a new location. It would be the new url3